components/Tooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {useRef} from 'react';
+import styles from '../styles/lobby.module.scss';
+import {animateTooltip} from '../utils/animations';
+
+interface TooltipProps {
+  text: string;
+  children: React.ReactNode;
+}
+
+function Tooltip({text, children}: TooltipProps): JSX.Element {
```

Comment:
I'm all for making things from scratch.. BUT I'm also a practical man.. why do this instead of using smth like [react-tooltip](https://www.npmjs.com/package/react-tooltip)? Also I think a better solution is similar to react-tooltip, where you have a "global" tooltip component and then `data props` for the elements are what trigger the tooltip.

Reply:
im cool with keeping it but these are just my thoughts